gateway/src/clob/clob.requests.ts

```diff
@@ -70,12 +70,29 @@ export interface ClobGetOrderResponse {
     | [];
 }
 
-export interface ClobPostOrderRequest extends ClobOrderbookRequest {
-  address: string;
-  side: Side;
-  orderType: OrderType;
+export interface CreateOrderParam {
   price: string;
   amount: string;
+  orderType: OrderType;
+  side: Side;
+  market: string;
+}
+
+export interface ClobPostOrderRequest
+  extends NetworkSelectionRequest,
+    CreateOrderParam {
+  address: string;
+}
+
+export interface ClobDeleteOrderRequestExtract {
+  market: string;
+  orderId: string;
+}
```

Comment:
Is there a reason for this naming convention over using `CancelOrderParam` like what we have for `CreateOrderParam`?

Reply:
No much reason. That could also work. Just trying to keep reference to `ClobDeleteOrderRequest` so it can be obvious that it's peculiar to CLOB.